manifests/charts/gateways/istio-ingress/templates/meshexpansion.yaml

```diff
@@ -16,7 +16,16 @@ spec:
         name: tcp-istiod
       hosts:
         - "*"
+    {{- if  .Values.global.centralIstiod }}
```

Comment:
Don't we already have a gateway on port 443 ? Can we use that instead ? 

This feature is not specific to centralIstiod - I'm not even certain that we need a special setting or mode for centralIstiod.

Reply:
Not built in, it is the cluster-aware-gateway which is deployed manually by the user.

Reply:
Sure - but there is one ingress gateway deployment where those configs are attached. 

So user will have to install the ingress we have - with some options. And if he wants to run a real ingress gateway in that cluster - which is a good idea - 443 will be messed up.